recipe/meta.yaml

```
@@ -1,31 +1,36 @@
-{% set version = "0.6.0" %}
+{% set version = "0.6.2" %}
 
 package:
   name: parse_type
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/p/parse_type/parse_type-{{ version }}.tar.gz
-  sha256: 20b43c660e48ed47f433bce5873a2a3d4b9b6a7ba47bd7f7d2a7cec4bec5551f
+  sha256: 79b1f2497060d0928bc46016793f1fca1057c4aacdf15ef876aa48d75a73a355
 
 build:
-  number: 1
-  noarch: python
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 0
+  skip: true  # [py37]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
-    - python >=3.6
+    - python
     - setuptools
     - pip
+    - wheel
   run:
-    - python >=3.6
+    - python
     - parse >=1.6
     - six
```

```suggestion
    - six >=1.15
```
https://github.com/jenisys/parse_type/blob/v0.6.2/pyproject.toml#L49